Virtual Librarian/Virtual Librarian/App.config

```diff
@@ -1,5 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+    <configSections>
+    </configSections>
+    <connectionStrings>
+        <add name="Virtual_Librarian.Properties.Settings.libraryDBConnectionString"
+            connectionString="Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=&quot;C:\Users\giriukaralius\source\repos\Virtual-Librarian\Virtual Librarian\Virtual Librarian\libraryDB.mdf&quot;;Integrated Security=True"
```

Comment:
don't we need to use a more relative path here? 'C:\Users\giriukaralius\source\repos\Virtual-Librarian\Virtual Librarian\Virtual Librarian\libraryDB.mdf'